Chap_Revisions.tex

```diff
@@ -1291,4 +1291,40 @@ \subsection{Added Attributes (Provisional)}
 \pasteAttributeItem{PMIX_STORAGE_ACCESS_TYPE}
 
 
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%%%%%%%%%% History: Version 4.2
+\section{Version 4.2: TBD}
+
+The v4.2 update includes the following changes from the v4.1 document:
+
+\subsection{Added Functions (Provisional)}
+
+\begin{compactitemize}
+  \item \refapi{PMIx_Value_load}
+  \item \refapi{PMIx_Value_unload}
+  \item \refapi{PMIx_Value_xfer}
+  \item \refapi{PMIx_Info_list_start}
+  \item \refapi{PMIx_Info_list_add}
+  \item \refapi{PMIx_Info_list_xfer}
+  \item \refapi{PMIx_Info_list_convert}
+  \item \refapi{PMIx_Info_list_release}
+  \item \refapi{PMIx_Topology_destruct}
+\end{compactitemize}
+
+\subsection{Deprecated attributes}
+
+The following macros were deprecated in v4.0:
+
+\begin{compactitemize}
+  \item \declaremacroDEP{PMIX_INFO_LIST_START}
+  \item \declaremacroDEP{PMIX_INFO_LIST_ADD}
+  \item \declaremacroDEP{PMIX_INFO_LIST_XFER}
+  \item \declaremacroDEP{PMIX_INFO_LIST_CONVERT}
+  \item \declaremacroDEP{PMIX_INFO_LIST_RELEASE}
+  \item \declaremacroDEP{PMIX_TOPOLOGY_DESTRUCT}
+  \item \declaremacroDEP{PMIX_TOPOLOGY_FREE}
```

Comment:
Only suggestion I would have here is to follow each deprecation item with a `replaced by the \refapi{...} function`

Reply:
Done - also searched out and replaced all references to the macros with respective APIs.